test/test_instance_types_service.py

```diff
@@ -102,7 +102,7 @@ def test_instance_types_get(self):
             self.provider,
             "instance_type")
         inst_type = self.provider.compute.instance_types.find(
-            name=instance_type_name)[0]
```

Comment:
@baizhang Throughout CloudBridge, the find method returns an array, since multiple matches are possible during a find. The get method in contrast, was intended for retrieving a single instance. Unfortunately, the docstring for the InstanceTypesService.find() method is incorrect, sorry about that, and states that the find() method returns an instance, instead of a list. Therefore, I think this should be fixed in the docstring instead.